samples/language/Comments.qs

```diff
@@ -0,0 +1,21 @@
+/// # Sample
+/// Comments
+///
+/// # Description
+/// Comments begin with two forward slashes (//) and continue until the
+/// end of line. Such end-of-line comments may appear anywhere in the
+/// source code. Q# does not currently support block comments.
+/// Documentation comments, or doc comments, are denoted with three
+/// forward slashes instead of two.
```

Comment:
```suggestion
/// Comments begin with two forward slashes (`//`) and continue until the
/// end of line. Comments may appear anywhere in the source code. 
/// Q# does not currently support block comments.
/// Documentation comments, or doc comments, are denoted with three
/// forward slashes (`///`) instead of two.
```

Reply:
Some notes on the suggestions:
- Text that is meant to be Q# source code can be enclosed between backticks ` `.
- Simplified one sentence to make it clearer (IMO).